grammars/ruby.cson

```diff
@@ -2285,6 +2310,35 @@
         'name': 'comment.line.number-sign.ruby'
       }
     ]
+  'separators':
+    'patterns': [
+      {
+        'match': '=>'
+        'name': 'punctuation.separator.key-value.ruby'
+      }
+      {
+        'match': ';'
+        'name': 'punctuation.separator.statement.ruby'
+      }
+      {
+        'match': ','
+        'name': 'punctuation.separator.object.ruby'
```

Comment:
Generally tokenized as `punctuation.separator.delimiter`.  Is `,` only for objects in Ruby?

Reply:
There are also commas in hashes so `punctuation.separator.delimiter` is indeed better.